github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```
@@ -931,6 +931,34 @@ presubmits:
             memory: 4Gi
         securityContext:
           privileged: true
+  - always_run: false
+    annotations:
+      fork-per-release: "true"
+      testgrid-dashboards: kubevirt-presubmits
+    cluster: prow-arm64-workloads
+    decorate: true
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 2h0m0s
+    labels:
+      preset-bazel-unnested: "false"
+    name: pull-kubevirt-unit-test-arm64
+    skip_branches:
+    - release-\d+\.\d+
+    spec:
```

Let's first not report to GitHub to eventually avoid confusion for users.

```suggestion
    skip_report: false
    spec:
```